examples/gno.land/r/gnoland/hof/gno.mod

```diff
@@ -0,0 +1,8 @@
+module gno.land/r/gnoland/hof
```

Comment:
It shouldn't be "gnoland"; it's not "official" or "useful" enough, and it allows too much permissionless activity. 

Let's make it a demo.

Reply:
Moved it to `gno.land/r/demo/hof`